apis/core.oam.dev/common/types.go

```diff
@@ -219,8 +219,10 @@ const (
 
 // ApplicationComponentStatus record the health status of App component
 type ApplicationComponentStatus struct {
-	Name string `json:"name"`
-	Env  string `json:"env,omitempty"`
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+	Cluster   string `json:"cluster"`
+	Env       string `json:"env,omitempty"`
```

Comment:
does this field still used?

Reply:
@wonderflow  For env-binding it is used. I think we can remain it in v1.3, deprecate in v1.4, remove it in v1.5.

Reply:
sure， that's fine for me